sycl/test-e2e/Regression/exclusive-scan-char-short.cpp

```diff
@@ -6,6 +6,7 @@
 // the maximum or minimum operator is computed correctly.
 #include "../helpers.hpp"
 #include <numeric>
+
 #include <sycl/sycl.hpp>
```

Comment:
Are we keeping `sycl/sycl.hpp` in this test?

Reply:
Yes, we have `"../helpers.hpp"` include here that has another `<sycl/sycl.hpp>` inside it. I've decided to leave that top-level file to the last rounds of cleanups, my current series is focused on local one directory at a time changes.